tesler-core/src/main/java/io/tesler/core/metahotreload/controller/MetaHotReloadController.java

```diff
@@ -0,0 +1,58 @@
+/*-
+ * #%L
+ * IO Tesler - Core
+ * %%
+ * Copyright (C) 2018 - 2019 Tesler Contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package io.tesler.core.metahotreload.controller;
+
+import io.tesler.core.metahotreload.service.MetaHotReloadService;
```

Comment:
These classes relate to https://github.com/tesler-platform/tesler/pull/146 , right?

Reply:
fixed